core/src/test/kotlin/fr/sncf/osrd/stdcm/StandardAllowanceTests.kt

```diff
@@ -330,7 +330,7 @@ class StandardAllowanceTests {
         occupancyTest(res, occupancyGraph, TIME_STEP)
         val thirdBlockEntryTime = (res.departureTime
                 + res.envelope.interpolateTotalTime(11000.0))
-        Assertions.assertEquals(1000.0, thirdBlockEntryTime, 2 * TIME_STEP)
+        Assertions.assertEquals(1000.0, thirdBlockEntryTime, 3 * TIME_STEP) // Actually 3 allowances are built
```

Comment:
Sorry, I got confused the other day, that's not actually what happens:

There's one potential timestep diff for the engineering allowance
Another one for the standard allowance (over several ranges but it shouldn't build up)
And a final one is added as margin when avoiding the conflict with the other train